eldritch/encounters.py

```diff
@@ -144,11 +144,105 @@ def Society4(char):
   cond = events.Conditional(char, check, "successes", {0: events.Nothing(), 2: skill})
   return events.Sequence([check, cond], char)
 
+def Administration1(char):
+  check = events.Check(char, "lore", -1)
+  dollars = events.Gain(char, {"dollars": 5})
+  return events.PassFail(char, check, dollars, events.Nothing())
+def Administration2(char):
+  return events.Gain(char, {"clues": 1})
+def Administration3(char):
+  check =  events.Check(char, "will", -1)
+  retainer = events.StatusChange(char, "retainer")
+  return events.PassFail(char, check, retainer, events.Nothing())
+def Administration4(char):
+  check = events.Check(char, "lore", -2)
+  spell = events.Draw(char, "spells", 1)
+  curse = events.Curse(char)
+  assist =  events.PassFail(char, check, spell, curse)
+  return events.BinaryChoice(char,"Help the professor and his students?", "Yes", "No", assist, events.Nothing())
```

Comment:
I have decided on a 100 char line limit. Suggested:
```suggestion
  return events.BinaryChoice(
      char,"Help the professor and his students?", "Yes", "No", assist, events.Nothing())
```